### DISCUSSION TITLE
Grafana Panel Threshold based multiple notification channel alerting

### DISCUSSION THREAD

**Discussion by Helloworld-can** [grafana/grafana#46552] [Alerting]
Hello All,

i'd like to request if a feature within Grafana single panel which would allow Alerts to be directed to a specific channel based on it's Threshold for example:

If CPU Usage is 70% Then Notification should only be sent to Slack Channel
If CPU Usage is 90% Then Notification should be sent to Slack Channel & PagerDuty

Right now the only option is to create multiple panels and then define alerts based on threshold and channel. This approach makes it messy and has poor impact on Performance where an end user has to see multiple dashboards for one use case.

**Comment by torkelo** [comment]
in the new alerting engine (you have to enable in ini file): https://grafana.com/docs/grafana/latest/alerting/unified-alerting/ 

You can create notification polices, and route alerts based on labels 

You would still need two alert rules, as there is only one firing state. But you can define in the rules if it's warn or criticial rule and set up generic notification rules

**Reply by Helloworld-can** [reply]
Thanks! It seems our organization requires an update of Grafana version to have this implemented. I'll try this in my own setup to validate.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
